sdk/common/index.ts

```diff
@@ -10,6 +10,10 @@ import { layoutItems, UniversalAddress, VAA } from '@wormhole-foundation/sdk-def
 export type VaaMessage = VAA<'TokenBridge:TransferWithPayload'>;
 export type TbrV3Payload = LayoutToType<typeof TBRv3Message>;
 
+export function throwError(message: string): never {
```

Comment:
funny wat to just avoid writting `new`, hehe...
isn't it more work to import `throwError` in different places than just writting `throw new Error`? or am I missing something :thinking: ?

Reply:
It's to use it as an expression, like:

```ts
const foo = someCall() ?? throwError('Missing foo');
```

Instead of the verbose:
```ts
const foo = someCall();
if (foo == null) {
    throw new Error('Missing foo');
}
```